### PR TITLE
Cleanup github actions

### PR DESCRIPTION
**Pull request by evrardjp**
This was a little bit hard to follow, so I wanted to clarify those with the current state of master branch.

Please do not merge, it's just a basis for discussion.

### CODE REVIEW

File: .github/workflows/periodics-daily.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue was automatically considered stale due to lack of activity. Please update it and/or join our slack channels to promote it, before it automatically closes (in 7 days).'
+        stale-pr-message: 'This PR was automatically considered stale due to lack of activity. Please refresh it and/or join our slack channels to highlight it, before it automatically closes (in 7 days).'
+        stale-issue-label: 'no-issue-activity'
+        stale-pr-label: 'no-pr-activity'
+        exempt-issue-labels: 'good-first-issue'
+  # This needs all the previous artifacts to be created:
+  # the image should be published, the repo should be tagged, the helm
+  # chart pushed. This ensure that the latest code merged didn't start to break from a day to another.
+  smoke-tests:
```
Comment (evrardjp) [inline .github/workflows/periodics-daily.yaml:25]: this can be removed from periodics if we accept this kind of tests on PRs (which I think is best)

**Comment by evrardjp** [comment]
Look at all this green! and they all run in under 30 minutes (23m for the longest). 

I will retrigger testing to see if I am unlucky.

I think we can go ahead with this, as it's cleaner to understand and see the scope of the tests. Of course, the change might have introduced regressions/problems in tests that are _not_ covered by this PR, but the only way to see and fix those, is to meet them in the wild.

**Comment by evrardjp** [comment]
Iteration number 2: Success, slowest run in 21 minutes.

**Review by dholbach — APPROVED** [review]
Nicely done!
